src/pages/workspace/WorkspaceInitialPage.js

```diff
@@ -24,9 +22,6 @@ import withFullPolicy, {fullPolicyPropTypes, fullPolicyDefaultProps} from './wit
 import * as PolicyActions from '../../libs/actions/Policy';
 
 const propTypes = {
-    /** Whether the current screen is focused. */
-    isFocused: PropTypes.bool.isRequired,
-
     ...fullPolicyPropTypes,
     ...withLocalizePropTypes,
     ...windowDimensionsPropTypes,
```

Comment:
Correct me if I'm wrong, but I believe this & related imports can be removed too

Reply:
We can get rid of windowDimensions for sure.
But we need the policy and localize prop types. (Used for props.translate, props.policy)

Reply:
Done!

Reply:
awesome thanks!